index.md

```diff
@@ -9,55 +9,191 @@ iOS and Android from a single codebase.
- for developing Flutter apps.
+![Shrine, iOS left and Android right](/images/whatisflutter/shrine_both.png)
 
-## Developer Guides
+This is an online marketplace app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo), 
+a showcase of Flutter apps to download and run on a phone. Shrine has 
+high-quality scrolling images, interactive cards, buttons and dropdown lists, 
+and a shopping cart page. Run Shrine on an iPhone or Android phone, then check
+out the single codebase for this and more examples [here](https://github.com/flutter/flutter/tree/master/examples) 
+in our github repository.
+
+No mobile development experience is required to get started. Apps are written 
+in [Dart](https://dartlang.org), which will look familiar if you've used a 
+language like Java or JavaScript. Experience with object-oriented languages 
+is definitely helpful, but even non-programmers have made Flutter apps!
 
-See these topics to understand how the Flutter framework works.
+## Why use Flutter?
 
- - [Widgets overview](widgets)
- - [Gestures in Flutter](gestures)
- - [Animations in Flutter](animations)
- - [Using custom fonts](custom-fonts)
- - [Layered design (video)](https://www.youtube.com/watch?v=dkyY9WCGMi0)
- - [Rendering pipeline (video)](https://www.youtube.com/watch?v=UUfXWzp0-DU)
- - [Dealing with box constraints in Flutter](layout)
+What are some advantages of Flutter? It helps you:
 
-## How-Tos
-
-Learn how to accomplish specific development tasks with Flutter.
-
- - [Building Layouts in Flutter](/tutorials/layout/)
- - [Adding Assets and Images](assets-and-images)
- - [Testing Flutter Apps](testing)
- - [Debugging Flutter Apps](debugging)
- - [Developing apps in the IntelliJ IDE](intellij-ide)
- - [Formatting Flutter code](formatting)
- - [Upgrading Flutter](upgrading)
- - [Accessing Platform and Third-Party Services](platform-services)
- - [Reading and Writing Files](reading-writing-files)
- - [Handling Text Input](text-input)
-
-## Helpful Resources
-
-More resources for Flutter developers.
-
- - [Source on GitHub](https://github.com/flutter/flutter)
- - [Architecture diagram](https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit?usp=sharing)
- - [Issue tracker](https://github.com/flutter/flutter/issues)
- - [Writing useful bug reports](bug-reports)
- - [FAQ](faq)
- - [Design principles](design-principles)
-
-## Support
+*   Be highly productive
+    *   Develop for iOS and Android from a single codebase
+    *   Do more with less code, even on a single OS, with a modern, expressive 
+language and a declarative approach
+    *   Prototype and iterate easily
+        *   Experiment by changing code and reloading as your app runs (with hot reload)
+        *   Fix crashes and continue debugging from where the app left off
+*   Create beautiful, highly-customized user experiences
+    *   Leverage a complete set of self-describing Material Design widgets
```

Comment:
I called them a "rich" set of widgets, since 'robust' has become cliche'd and with "near-complete", it sounds kind of unfinished. And honestly, they're cool and great.